### ISSUE TITLE
Category bottomline is clickable but does not link to product page

### ISSUE THREAD

**Issue by andrew-ebsco** [YotpoLtd/magento2-module-yotpo-reviews#269]
We have enabled the category bottomline so customers can see the number of stars and reviews when browsing the product list. If you hover over the stars or number of reviews, the cursor changes to a hand like it is clickable, however, clicking it does nothing. I submitted a support request to Yotpo, but was told we have to add the link ourselves. Since the bottomline is generated by this module, the link should be added in it. Looks like it will need to be added to the `Plugin/AbstractYotpoReviewsSummary.php` under `_getCategoryBottomLineHtml`. Let me know if I can provide any more info.

**Comment by pinchas13** [comment]
Hey @andrew-ebsco 
Thank you for reaching out.
Can you please share the URL where the issue happens?

**Comment by andrew-ebsco** [comment]
Yes [this page](https://www.moultriefeeders.com/products/moultrie-mobile) is a good example. You can see the stars and number of reviews on the product card. If you place your cursor over them, the text has a hover state and changes the cursor, but when you click on it, nothing happens. We expect that clicking the stars would navigate to reviews on the product page.

**Comment by pinchas13** [comment]
hey @andrew-ebsco
By default, our CSS settings for the star rating are that it will show the 'hand' cursor when hovering.
I was able to fix the hover issue in the category page via the CSS editor in your Yotpo account.
Check the changes In the Yotpo admin > On-site widgets > Widget CSS editor > scroll to the bottom.

If you wish to add 'a' tag to the star rating code, the path is 
magento2-module-yotpo-reviews/view/frontend/templates/bottomline.phtml

Please reach out to us by submitting a support ticket from the Yotpo admin if any additional help is needed.
Have a great day!

**Comment by andrew-ebsco** [comment]
Hey @pinchas13, thanks for the help. However, we actually want to make the category stars link to the product page. Changing the bottomline.phtml template only affects the product page, not category pages. It would be nice to have an option to use a link in the `_getCategoryBottomLineHtml` function.
